flow/Platform.cpp

```diff
@@ -2841,7 +2841,8 @@ void setupSlowTaskProfiler() {
 #endif
 }
 
-#ifdef __linux__
+#if defined(__linux__)
```

Comment:
#if defined(__linux__) && !defined(__clang__)

Reply:
And if you could add a `// FIXME: Delete this when official build environment is upgraded from Ubuntu 14.04.`, that'd be great :)